sdcm/cluster.py

```diff
@@ -1621,8 +1621,22 @@ def get_ldap_config():
                 'ldap_bind_dn': f'cn=admin,{LDAP_BASE_OBJECT}',
                 'ldap_bind_passwd': LDAP_PASSWORD}
 
+    @staticmethod
+    def get_saslauthd_config():
+        ldap_server_ip = '127.0.0.1' if IP_SSH_CONNECTIONS == 'public' or Setup.MULTI_REGION else Setup.LDAP_ADDRESS[0]
+        ldap_port = LDAP_SSH_TUNNEL_LOCAL_PORT if IP_SSH_CONNECTIONS == 'public' or Setup.MULTI_REGION else \
+            Setup.LDAP_ADDRESS[1]
+        return {'ldap_server': f'ldap://{ldap_server_ip}:{ldap_port}/'
+                'ldap_seach_base': LDAP_BASE_OBJECT,
+                'ldap_bind_dn': f'cn=admin,{LDAP_BASE_OBJECT}',
+                'ldap_bind_pd': LDAP_PASSWORD}
+
     def create_ldap_users_on_scylla(self):
         self.run_cqlsh(f'CREATE ROLE \'{LDAP_ROLE}\' WITH SUPERUSER=true')
+        for user in LDAP_USERS:
+            if user == 'cassandra':
+                # User `cassandra' has been created by default
+                continue
```

Comment:
This is not good anyway, if we want to use anywhere "Cassandra" user, let's have another list of users like "BUILTIN_USERS".

Reply:
fixed.